h/assets.yaml

```diff
@@ -195,6 +195,7 @@ inject:
 
 inject_js:
   contents:
+  - url
```

Comment:
I'm not surprised, but I still think this is the right thing to do. For one, there's a symmetry argument: I don't want to have to think about or justify why some but not other polyfills are bundled and others are checked/loaded. For another, there's the argument for not bundling things you don't need, however small the footprint happens to be. We have a pattern here, please follow it.